app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -2,7 +2,7 @@ module PhysicalServerHelper::TextualSummary
   def textual_group_properties
     TextualGroup.new(
       _("Properties"),
-      %i(name model product_name manufacturer machine_type serial_number ems_ref capacity memory cores health_state loc_led_state)
+      %i(name model product_name manufacturer machine_type serial_number ems_ref capacity memory cores network_devices health_state loc_led_state)
```

Comment:
Specify `network_devices` in a new Textual Group - `Relationships`

```rb
def textual_group_relationships
  TextualGroup.new(_("Relationships"), %i(network_devices))
end
```

Reply:
Actually, on second thoughts, displaying it as 'Properties'  (the way you have it currently), should be fine.